src/main/java/Deadline.java

```diff
@@ -0,0 +1,27 @@
+public class Deadline extends Task{
```

Comment:
As per coding standards, the class name and "{" should be separated by a space. Eg "...extends Task {...". This also applies to while, for loops, etc.